App.Goods/Services/IValidateService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace App.Goods.Services
+{
+    public interface IValidateService
```

Comment:
Reanme validate service to normalize service. Because it does perform validation actually